src/main/resources/META-INF/resources/webjars/service/prov/prov.js

```diff
@@ -1497,33 +1506,32 @@ define(['sparkline', 'd3'], function () {
 		let details = "";
 		let totalDuration = 0;
 		let dataPoints = [];
-		if (input_workload.length > 1) {
-			for (let i = 1; i < input_workload.length; i++) {
+		if (input_workload.length > 1) {		
+			for ( let i =0; i < input_workload.length; i++) {
 				let data = input_workload[i].split('@');
-				if (data.length == 2) {
+				if (data.length == 2 && data[0]!=="") {
 					totalDuration = totalDuration + parseInt(data[0]);
-					if (totalDuration <= 100 && data[0] != ("" || 0) && data[1] != "") {
+					if (data[0] !== 0 && data[1] !== "") {
 						workload = workload + (data[0] * data[1] / 100);
 						details = details + `,${data[0]}@${data[1]}`
 						dataPoints.push({ "duration": data[0], "cpu": data[1] });
 					}
-				}
+				}	
 			}
-			if (workload != 0) {
-				_('instance-workload').val(workload + details);
-			} else {
+			if (workload == 0) {
 				_('instance-workload').val(input_workload[0]);
+			} else {
+				_('instance-workload').val(Math.round(workload) + details);
```

Comment:
String format should be used, as at line 1492. Maybe a common function can be written.